agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/profiler/AlternativeJfrConfigurations.java

```diff
@@ -16,8 +16,9 @@
 import org.slf4j.LoggerFactory;
 
 /** Allows loading alternative jfc configuration files. */
-public class AlternativeJfrConfigurations {
-  private static final Logger LOGGER = LoggerFactory.getLogger(AlternativeJfrConfigurations.class);
+class AlternativeJfrConfigurations {
+
+  private static final Logger logger = LoggerFactory.getLogger(AlternativeJfrConfigurations.class);
```

Comment:
ClientLogger?

Reply:
ya, probably, let's do this in a separate PR across the whole codebase